src/Discord.Net.Rest/Entities/Guilds/RestGuild.cs

```diff
@@ -176,6 +176,11 @@ public Task<RestGuildChannel> GetChannelAsync(ulong id, RequestOptions options =
             var channels = await GuildHelper.GetChannelsAsync(this, Discord, options).ConfigureAwait(false);
             return channels.Select(x => x as RestVoiceChannel).Where(x => x != null).ToImmutableArray();
         }
+        public async Task<IReadOnlyCollection<RestGuildChannelCategory>> GetChannelCategoriesAsync(RequestOptions options = null)
+        {
+            var channels = await GuildHelper.GetChannelsAsync(this, Discord, options).ConfigureAwait(false);
+            return channels.Select(x => x as RestGuildChannelCategory).Where(x => x != null).ToImmutableArray();
```

Comment:
Opt for `ToReadOnlyCollection()` when you have a collection with a `.Count` property, rather using `.ToImmutableArray()`.

Reply:
Only did the same as what was done for the methods right above it (GetVoiceChannelsAsync and GetTextChannelsAsync)

Reply:
Ah. Couldn't see them from the review page.